docs/src/main/asciidoc/kubernetes-resources.adoc

```diff
@@ -139,3 +139,36 @@ By default the Kubernetes resources do not contain readiness and liveness probes
 
 The values of the generated probes will be determined by the configured health properties: `quarkus.smallrye-health.root-path`, `quarkus.smallrye-health.liveness-path` and `quarkus.smallrye-health.readiness-path`.
 More information about the health extension can be found in the relevant link:health-guide[guide].
+
+== Tuning the generated resources using application.properties
+
+The kubernetes extension allows tuning the generated manifest, using the `application.properties` file.
+Here are some examples:
+
+=== Changing the number of replicas:
+To change the number of replicas from 1 to  3:
+
+[source]
+----
+quarkus.kubernetes.replicas=3 
+----
+
+=== Adding labels:
+To add a new label to all generated resources, say `foo=bar`:
+
+[source]
+----
+quarkus.kubernetes.labels[0].key=foo
+quarkus.kubernetes.labels[0].value=bar
+----
+
+=== Customizing the readiness probe:
+To set the initial delay of the probe to 20 seconds and the period to 45:
+
+[source]
+----
+quarkus.kubernetes.readinessProbe.initialDelaySeconds=20
+quarkus.kubernetes.readinessProbe.periodSeconds=45
```

Comment:
This doesn't follow the case convention of `application.properties`. I know why you did that but this is an issue.

There might be some config util that you could use to transform dash case to camel case in the config stuff.

Reply:
This extension is not leveraging the Quarkus config to read these properties. It's `dekorate` itself that parses the configuration.
@iocanel would it be possible to use dash/kebab/whatever-it's-called case instead of camel case?

Reply:
To be honest, I would prefer the dash over camel case myself. I will have a look at it.

Reply:
So, we have the opposite operation in `io.quarkus.runtime.util.StringUtil` but I don't see anything for the other way around.

Reply:
If I am not mistaken,@iocanel took care of the problem in dekorate so I think we can expect a PR update soon

Reply:
Add that in dekorate 0.9.1